src/PostgisRegisterTypesPlugin.ts

```diff
@@ -74,31 +82,95 @@ const plugin: Plugin = builder => {
                   type: GeoJSON,
                   description: "Converts the object to GeoJSON",
                 },
+                srid: {
+                  type: GraphQLInt,
+                  description: "Spatial reference identifier (SRID)",
+                },
               },
               resolveType(value: any, _info?: GraphQLResolveInfo) {
-                const subtype = SUBTYPE_BY_PG_GEOMETRY_TYPE[value.__gisType];
                 const Type =
                   constructedTypes[type.id] &&
-                  constructedTypes[type.id][subtype];
+                  constructedTypes[type.id][value.__gisType];
                 return Type;
               },
-              description:
-                "All PostGIS geography types implement this interface",
+              description: `All ${type.name} types implement this interface`,
             },
             {
-              isPgGISGeographyInterface: true,
+              isPgGISInterface: true,
             }
           );
+          // Force creation of all GraphQL types that could be resolved from this interface
+          const subtypes: Array<Subtype> = [1, 2, 3, 4, 5, 6, 7];
+          for (const subtype of subtypes) {
+            for (const hasZ of [false, true]) {
+              for (const hasM of [false, true]) {
+                const typeModifier = getGISTypeModifier(subtype, hasZ, hasM, 0);
+                const Type = getGisType(type, typeModifier);
+                includeType(Type);
+              }
+            }
+          }
```

Comment:
This is the most sensible way of solving this problem and loop performance is not a concern for these 28 iterations. 👍